extra/changelog-generator/test-changelog-generator

```diff
@@ -292,9 +294,9 @@ TREE=$(git cat-file commit HEAD | grep '^tree ' | sed -e 's/^tree //')
 git reset --hard $(git commit-tree -p $(git rev-parse HEAD) $TREE <<EOF
 Changelog with Windows line endings N66
 
-Changelog: title
-
-stuff stuff
+Changelog: title
+
+stuff stuff
```

Comment:
What? Was there some invisible characters here or something

Reply:
nvm, carriage return at the end of this lines originally. Weird that vscode removed these when simply opening the file...